### PR TITLE
fix(statusline): adjust to the new API (query)

### PR DESCRIPTION
**Pull request by ranjithshegde**
Just a quick fix before the refactor recommended by @clason

### CODE REVIEW

File: lua/nvim-treesitter.lua [hunk 1]
```diff
@@ -48,15 +49,16 @@ function M.statusline(opts)
...
+  --   options = { indicator_size = opts }
+  -- end
+  local bufnr = options.bufnr or 0
   local indicator_size = options.indicator_size or 100
   local type_patterns = options.type_patterns or { "class", "function", "method" }
   local transform_fn = options.transform_fn or transform_line
   local separator = options.separator or " -> "
 
-  local current_node = ts_query.get_node_at_cursor()
+  local current_node = ts_utils.get_node_at_cursor()
```
Comment (clason) [inline lua/nvim-treesitter.lua:61]: That function is deprecated; please use the upstream version!

**Review by clason — APPROVED** [review]

File: lua/nvim-treesitter.lua (hunk 1 above)
Comment (clason) [inline lua/nvim-treesitter.lua:61]: oops...

**Review by clason — APPROVED** [review]

File: lua/nvim-treesitter.lua (hunk 1 above)
Comment (ranjithshegde) [reply lua/nvim-treesitter.lua:61]: I could find the `get_node_text()` upstream but I just cant find the `get_node_at_cursor() `upstream. I am on the latest build of nvim

Comment (clason) [reply lua/nvim-treesitter.lua:61]: Yeah, that was my bad. I got a bit carried away there. (This _should_ be upstreamed, but clearly hasn't yet.)

**Pull request merged**